### DISCUSSION TITLE
Busco grupo para  Hackaton

### DISCUSSION THREAD

**Discussion by Cristoper-ds** [Launch-X-Latam/MisionFrontEnd#528] [Team Posts]
Hola a todos, me gustaría unirme a un grupo o crear uno, pueden enviarme un mensaje al correo
lx07860@innovaccion.mx

**Comment by NArAr12** [comment]
Hola, me llamo Natalia quisiera formar parte de tu equipo.
Mi correo lx10738@innovaccion.mx

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu metanse aqui y comienzan a buscar para no tener
spam
